### PR TITLE
Add EntryPoint public testnet

### PR DESCRIPTION
**Pull request by migueldingli1997**

### CODE REVIEW

File: testnets/entrypointtestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      ],
+      "base": "uentry",
+      "name": "EntryPoint",
+      "display": "entry",
+      "symbol": "ENTRY",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/entrypointtestnet/images/entry.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/entrypointtestnet/images/entry.svg"
+      },
+      "coingecko_id": "entrypoint",
```
Comment (JeremyParish69) [inline testnets/entrypointtestnet/assetlist.json:27]: Please remove the coingecko id

File: testnets/entrypointtestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/entrypoint-zone/testnets",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": [
+      "v1.0.0"
+    ],
+    "cosmos_sdk_version": "0.47.4",
+    "consensus": {
```
Comment (JeremyParish69) [inline testnets/entrypointtestnet/chain.json:40]: Can you please copy to consensus and sdk version into the first object in the 'version' array?

File: testnets/entrypointtestnet/assetlist.json (hunk 1 above)
Comment (migueldingli1997) [inline testnets/entrypointtestnet/assetlist.json:27]:
```suggestion
      },
```

File: testnets/entrypointtestnet/chain.json (hunk 2 above)
Comment (migueldingli1997) [reply testnets/entrypointtestnet/chain.json:40]: Sorry I don't understand exactly @JeremyParish69. I already have consensus and sdk version in the 'versions' array.

I copied the format used in the Osmosis testnet:
- Consensus and sdk version under 'codebase': https://github.com/cosmos/chain-registry/blob/master/testnets/osmosistestnet/chain.json#L39-L42
- Consensus and sdk version under 'versions': https://github.com/cosmos/chain-registry/blob/master/testnets/osmosistestnet/chain.json#L56-L60

File: testnets/entrypointtestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      "kind": "ping.pub",
+      "url": "https://explorer.entrypoint.zone/entrypoint",
+      "tx_page": "https://explorer.entrypoint.zone/entrypoint/tx/${txHash}",
+      "account_page": "https://explorer.entrypoint.zone/entrypoint/account/${accountAddress}"
+    },
+    {
+      "kind": "ping.pub",
+      "url": "https://ping.pub/entrypoint",
+      "tx_page": "https://ping.pub/entrypoint/tx/${txHash}",
+      "account_page": "https://ping.pub/entrypoint/account/${accountAddress}"
```
Comment (migueldingli1997) [inline testnets/entrypointtestnet/chain.json:91]:
```suggestion
      "kind": "ping.pub",
      "url": "https://testnet.ping.pub/entrypoint",
      "tx_page": "https://testnet.ping.pub/entrypoint/tx/${txHash}",
      "account_page": "https://testnet.ping.pub/entrypoint/account/${accountAddress}"
```

**Pull request merged**